backend/models/comment.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require("mongoose");
+
+var commentSchema = new mongoose.Schema(
+    {
+        lastReviewed: {
+        type: String
+        },
+        value: {
+        type: String
+        },
+        userId: {
```

Comment:
I think it might be a good idea to also include user email in the schema, IK it came up a lot with Application comments. Also, do you think it might be useful to index on userId? I feel like that might be what we use to query a lot (this is just from working with app comments)

Reply:
Added Email
For indexing besides displaying them per decision canvas, where else do we query for comments?

Reply:
Okay, this will be a compound index, of either (appId, criteriaId) or (appId, criteriaId, userId), will do that, and update here